### ISSUE TITLE
Unable to load mockup pages in certain browsers

### ISSUE THREAD

**Issue by mraineri** [DMTF/Redfish-Mockup-Server#13]
When trying to access resources on the mockup server with Chrome 60.0.3112.101, Service Root is able to be access successfully, but subordinate resources seem to just "hang", ultimately causing an error in the browser. Firefox seems to work okay though. I do see a difference in the types of request headers being sent between the browsers, and I suspect one of those headers is not being handled nicely by the mockup server.

Good case (Firefox):
```
   GET: Headers: Host: localhost:8000
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64; rv:54.0) Gecko/20100101 Firefox/54.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate
Connection: keep-alive
Upgrade-Insecure-Requests: 1


False
127.0.0.1 - - [17/Aug/2017 15:57:13] "GET /redfish/v1/Managers HTTP/1.1" 200 -
```

Bad case (Chrome):
```
   GET: Headers: Host: localhost:8000
Connection: keep-alive
Cache-Control: max-age=0
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.101 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.8


False
127.0.0.1 - - [17/Aug/2017 15:59:41] "GET /redfish/v1/Managers HTTP/1.1" 200 -
```

Although the "bad" case returns a 200, the web browser shows:
```
This page isn’t working

localhost sent an invalid response.
ERR_INVALID_HTTP_RESPONSE
```

**Comment by mraineri** [comment]
Assigning to Bill and Tomas to debug

**Comment by mraineri** [comment]
Might be an issue with how headers are sent by the server (if headers in the mockup are captured).

**Comment by hohene** [comment]
if this helps, curl shows this error on the links for DELL mockups.
```
curl 127.0.0.1:8080/redfish/v1/Systems -v | python -m json.tool
*   Trying 127.0.0.1...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Connected to 127.0.0.1 (127.0.0.1) port 8080 (#0)
> GET /redfish/v1/Systems HTTP/1.1
> Host: 127.0.0.1:8080
> User-Agent: curl/7.47.0
> Accept: */*
> 
{ [522 bytes data]
100   522    0   522    0     0  45304      0 --:--:-- --:--:-- --:--:-- 52200
* Connection #0 to host 127.0.0.1 left intact
Extra data: line 1 column 18 - line 13 column 2 (char 17 - 522)
```

**Comment by mraineri** [comment]
Closing; fixed with #14
